src/schema/sources.ts

```diff
@@ -126,6 +128,11 @@ export const typeDefs = /* GraphQL */ `
     Logged-in member object
     """
     currentMember: SourceMember
+
+    """
+    Privileged members
+    """
+    privilegedMembers: [SourceMember]
```

Comment:
Noticed that by exposing this someone could craft a GQL query to our API (same is for `members` field that existed before) to get `referralToken` of other users. Is this security concern? @idoshamun

Reply:
Talked to Ido, we should log this to fix in another PR.

Reply:
Talked to Ido, we should fix this in another PR. Logged it here https://dailydotdev.atlassian.net/browse/WT-1216 FYI @rebelchris